test/dotnet-new.Tests/GivenThatIWantANewApp.cs

```diff
@@ -75,8 +75,7 @@ public void RestoreDoesNotUseAnyCliProducedPackagesOnItsTemplates()
         }
 
         [Theory]
-        [InlineData("console", "microsoft.netcore.app")]
```

Comment:
In order to re-enable this test, just update the MicrosoftNETCoreAppPackageVersion to 2.1.1 in DependencyVersions.props.

Reply:
I think it will need to be the latest pre-release version of 2.1.1 right?